### PR TITLE
Automatic cross-site static service discovery inside single cluster

### PR DESCRIPTION
**Pull request by dmvolod**

### CODE REVIEW

File: pkg/controller/constants/constants.go [hunk 1]
```diff
@@ -73,7 +73,8 @@ const (
 	NativeImageMarker         = "native"
 	GeneratedSecretSuffix     = "generated-secret"
 	InfinispanFinalizer       = "finalizer.infinispan.org"
-	SiteServiceTemplate       = "%v-site"
+	SiteServiceNameTemplate   = "%v-site"
+	SiteServiceFQNTemplate    = "%s.%s.svc.cluster.local"
```
Comment (ryanemerson) [inline pkg/controller/constants/constants.go:77]: Does this need to be a constant? It's only used once.

Comment (dmvolod) [reply pkg/controller/constants/constants.go:77]: Yes, exactly. I will move both `SiteServiceNameTemplate` and `SiteServiceFQNTemplate` to the `types_util.go`

**Review by ryanemerson — APPROVED** [review]

**Comment by oraNod** [comment]
@dmvolod Does this eliminate the need for users to specify any `spec.service.sites` configuration for cross-site in a single cluster?

**Comment by dmvolod** [comment]
> Does this eliminate the need for users to specify any `spec.service.sites` configuration for cross-site in a single cluster?

Yes, @oraNod correct question. I will update an examples,for static-internal in this case

**Review by pruivo — APPROVED** [review]

**Comment by ryanemerson** [comment]
Thanks @dmvolod

**Pull request merged**